packages/manager/modules/pci/src/projects/project/private-networks/private-networks.service.js

```diff
@@ -19,11 +25,40 @@ export default class {
         serviceName,
       })
       .$promise.then((privateNetworks) =>
-        sortBy(privateNetworks, 'vlanId').filter(
-          ({ status }) => !DELETING_STATUS.includes(status),
+        this.$q.all(
+          map(privateNetworks, (privateNetwork) =>
+            this.getSubnets(serviceName, privateNetwork),
+          ),
         ),
       );
   }
+  // TODO: getSubnets call need to trigger on expanding the rows for which we need to update expandable data grid under UI kit. currently calling it under getPrivateNetworks
+
+  getSubnets(serviceName, privateNetwork) {
+    return this.OvhApiCloudProjectNetworkPrivateSubnet.v6()
+      .query({
+        serviceName,
+        networkId: privateNetwork.id,
+      })
+      .$promise.then((data) => {
+        return {
+          ...privateNetwork,
+          subnet: [
+            ...data.map((subnet) => ({
+              ...subnet,
+              allocatedIp: subnet.ipPools
+                .map((ipPool) => `${ipPool.start} - ${ipPool.end}`)
+                .join(' ,'),
+              dhcp: subnet.ipPools
+                .map(
+                  (ipPool) => (ipPool.dhcp === true ? 'Active' : 'Suspended'), // TODO put active/inactive as constant
```

Comment:
**nitpick:** `Active` and `Suspended` needs a translation. So better to main as a boolean.